src/wp-admin/user-edit.php

```diff
@@ -733,6 +733,7 @@
 
 	</table>
 
+<?php if ( wp_is_application_passwords_available_for_user( $user_id ) || ! wp_is_application_passwords_supported() ) : // phpcs:disable Generic.WhiteSpace.ScopeIndent ?>
```

Comment:
To keep the changes focused on resolving the reported issue, `// phpcs:disable` is added. The indentation can be fixed in a separate follow-up commit.

Noting though that the original code before changeset https://core.trac.wordpress.org/changeset/51980 had the closing `<?php endif; ?>` where it is in this PR but the line 736 was indented by 2 tabs. Tried to revert the indentation to the original, but as commit https://github.com/WordPress/wordpress-develop/pull/2069/commits/fe6ae0c7746444203a32ae28dfcb3f2b03c3a594 shows, that produced more phpcs unhappiness.

I don't see the indentation reformatting as a blocker for resolving this issue by Beta 4. BUT the formatting should be resolved shortly after and the `// phpcs:disable` and `// phpcs:enable` annotations removed.

@costdev can you please open a new PR once this one is committed, i.e. so we don't forget to clean this up?